### ISSUE TITLE
`purified_bp` should be incremented before `if not succ:`

### ISSUE THREAD

**Issue by uchkw** [sfc-aqua/wqrn-tutorial#4]
In the last cell of both WQRN_hands_on.ipynb and WQRN_hands_on_ja.ipynb, `purified_bp` is incremented after the `if` condition, so if the `if` condition is met (i.e. the `is_purification_successful()` is not implemented correctly), `purified_bp` will not be incremented and the cell will run forever.

I think that `purified_bp` should be incremented before the if condition, as follows.
```python
while purified_bp < 1000:
    succ, res = is_purification_successful()
    purified_bp += 1
    if not succ:
        continue
```

**Comment by Naphann** [comment]
Good catch, I was also told this a few days ago and planning to push a fix in a few hours.

**Comment by Naphann** [comment]
This was solved with cd29495
